pandas/tests/series/indexing/test_indexing.py

```diff
@@ -188,12 +195,12 @@ def test_setitem_ambiguous_keyerror(indexer_sl):
 
 def test_setitem(datetime_series):
     datetime_series[datetime_series.index[5]] = np.NaN
-    datetime_series[[1, 2, 17]] = np.NaN
-    datetime_series[6] = np.NaN
-    assert np.isnan(datetime_series[6])
-    assert np.isnan(datetime_series[2])
+    datetime_series.iloc[[1, 2, 17]] = np.NaN
```

Comment:
isnt this test specifically for `__setitem__`?

Reply:
this is an outdated file, I didn't modify it - I presume you're commenting on the merge commit?